attest/verifier/config/src/lib.rs

```diff
@@ -0,0 +1,449 @@
+                assert_eq!(
+                    MRENCLAVE,
+                    &hex!("dca7521ce4564cc2e54e1637e533ea9d1901c2adcbab0e7a41055e719fb0ff9d")
+                );
+                assert_eq!(mitigated_config_advisories, &Vec::<String>::default());
+                assert_eq!(
+                    mitigated_hardening_advisories,
+                    &["INTEL-SA-00334", "INTEL-SA-00615"]
+                );
+            }
+            _ => panic!("unexpected"),
+        };
+
+        let v4 = tms.table.get("v4").unwrap();
+        assert_eq!(v4.len(), 4);
+        let v4_consensus = v4.get("consensus").unwrap();
+        match v4_consensus {
+            StatusVerifierConfig::Mrenclave {
+                MRENCLAVE,
+                mitigated_config_advisories,
+                mitigated_hardening_advisories,
+            } => {
+                assert_eq!(
+                    MRENCLAVE,
+                    &hex!("e35bc15ee92775029a60a715dca05d310ad40993f56ad43bca7e649ccc9021b5")
+                );
+                assert_eq!(mitigated_config_advisories, &Vec::<String>::default());
+                assert_eq!(
+                    mitigated_hardening_advisories,
+                    &["INTEL-SA-00334", "INTEL-SA-00615", "INTEL-SA-00657"]
+                );
+            }
+            _ => panic!("unexpected"),
+        };
+        let v4_fog_view = v4.get("fog-view").unwrap();
+        match v4_fog_view {
+            StatusVerifierConfig::Mrenclave {
+                MRENCLAVE,
+                mitigated_config_advisories,
+                mitigated_hardening_advisories,
+            } => {
+                assert_eq!(
+                    MRENCLAVE,
+                    &hex!("8c80a2b95a549fa8d928dd0f0771be4f3d774408c0f98bf670b1a2c390706bf3")
+                );
+                assert_eq!(mitigated_config_advisories, &Vec::<String>::default());
+                assert_eq!(
+                    mitigated_hardening_advisories,
+                    &["INTEL-SA-00334", "INTEL-SA-00615", "INTEL-SA-00657"]
+                );
+            }
+            _ => panic!("unexpected"),
+        };
+
+        let _ = tms.create_verifier("consensus").unwrap();
+        let _ = tms.create_verifier("fog-ingest").unwrap();
+        let _ = tms.create_verifier("fog-ledger").unwrap();
+        let _ = tms.create_verifier("fog-view").unwrap();
+
+        assert!(tms.create_verifier("impostor").is_err());
+    }
+
+    const TEST_DATA2: &str = r#"{
+    "v3": {
+       "consensus": {
+           "MRENCLAVE": "207c9705bf640fdb960034595433ee1ff914f9154fbe4bc7fc8a97e912961e5c",
+           "mitigated_config_advisories": ["FOO"],
+           "mitigated_hardening_advisories": ["INTEL-SA-00334", "INTEL-SA-00615"]
+       },
+       "fog-ingest": {
+           "MRSIGNER": "2c1a561c4ab64cbc04bfa445cdf7bed9b2ad6f6b04d38d3137f3622b29fdb30e",
+           "product_id": 1,
+           "minimum_svn": 4,
+           "mitigated_config_advisories": ["FOO"],
```

Comment:
:+1: Like the mixing of MRSIGNER and MRENCLAVE in the same json, and the addition of config advisories